### ISSUE TITLE
Adopt black as the formatting tool that pylint uses?

### ISSUE THREAD

**Issue by PCManticore** [PyCQA/pylint#2154]

**Comment by PCManticore** [comment]
Sounds good, thanks for the links! I imagine that first we have to run `black` over all the source files to convert them?

**Comment by asottile** [comment]
yep! pre-commit can help you do that as well `pre-commit run black --all-files` (once you've hooked up the configuration)
